packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx

```diff
@@ -90,7 +86,16 @@ export const ChatMessageComponentAsEditBox = (props: ChatMessageComponentAsEditB
   }, [theme.palette.themePrimary]);
 
   return (
-    <Stack className={mergeStyles(borderAndBoxShadowStyle(theme))}>
+    <Stack
+      className={mergeStyles(
+        borderAndBoxShadowStyle({
+          theme,
+          errorColor: isDarkThemed(theme) ? '#f1707b' : '#a80000',
```

Comment:
we added more than just theme. In this case are we doing special overrides that should just be in theme?

Reply:
I didn't want to change any of that logic in this PR, this just does the exact same logic that sendbox does to ensure consistency (maybe I should pull into into a component they both use..)
But ideally we should be using some theme-based semantic color that defines an error:
![image](https://user-images.githubusercontent.com/2684369/156416321-9dafd162-22ec-4a4b-b52a-7d89d9e5d03e.png)

Filed a bug for this: [Bug 2779691](https://skype.visualstudio.com/SPOOL/_workitems/edit/2779691): [Web] Can sendbox error coloring use a theme value?